addons/fortify/functions/fnc_registerObjects.sqf

```diff
@@ -30,7 +30,7 @@ if (_side isEqualTo sideUnknown) exitWith {ERROR_1("Bad Side %1",_this);};
 
 _objects select {
     private _isValid = _x params [["_xClassname", "", [""]], ["_xCost", 0, [0]]];
-    private _category = toLower (_x param [2, "", [""]]);
+    private _category = toLowerANSI (_x param [2, "", [""]]);
```

Comment:
Couldn't this contain non-ANSI chars?

Reply:
Given it's public and there are no limitations mentioned, I'm going to go with it can contain non-ANSI chars - reverting change.